src/genie/libs/parser/nxos/show_lag.py

```diff
@@ -328,9 +328,11 @@ def cli(self, output=None):
             r'(?P<bundle_id>[\d]+)[\xa0 ]+(?P<name>[\w\-]+)\((?P<flags>[\w]+)\)?[\xa0 '
             r']+(?P<type>\w+)[\xa0 ]+(?P<protocol>[\w\-]+)?[\xa0 ]+(?P<ports>[\w\-/() '
             r'\xa0]+ *)?$')
+        p2 = re.compile(
```

Comment:
Please have example output as comment for the change of p2 regex.

Reply:
added example output